vite.config.js

```diff
@@ -43,29 +44,17 @@ export default defineConfig(({ mode }) => {
       strictPort: true,
       cors: false,
       proxy: {
-        '/game': {
-          target: HOST_SERVER_URL,
-          changeOrigin: true,
-        },
-        // Required for the health response to work on the client
-        '/health': {
-          target: HOST_SERVER_URL,
-          changeOrigin: true,
-        },
-        '/user': {
-          target: HOST_SERVER_URL,
-          changeOrigin: true,
-        },
-        '/test': {
+        '/api': {
```

Comment:
Only need to proxy `/api/*` now in Vite since it's all grouped under the same namespace.